test/Confluent.Kafka.IntegrationTests/AdminClient_DescribeTopics.cs

```diff
@@ -0,0 +1,189 @@
+                    RequestTimeout = TimeSpan.FromSeconds(30), 
+                    IncludeAuthorizedOperations = false
+                };
+                var describeOptionsWithAuthOps = new Admin.DescribeTopicsOptions() { 
+                    RequestTimeout = TimeSpan.FromSeconds(30), 
+                    IncludeAuthorizedOperations = true
+                };
+
+                var nonexistTopic = Guid.NewGuid().ToString();
+                var topicList = new List<string>
+                {
+                    nonexistTopic,
+                    singlePartitionTopic
+                };
+
+                try
+                {
+                    var descResult = await adminClient.DescribeTopicsAsync(
+                        TopicCollection.OfTopicNames(topicList),
+                        describeOptionsWithTimeout);
+                }
+                catch (DescribeTopicsException ex)
+                {
+                    var resCount = ex.Results.TopicDescriptions.Count;
+                    Assert.Equal(2, resCount);
+                    
+                    // TODO: remove after fix
+                    ex.Results.TopicDescriptions.Sort(
+                        (a,b) =>
+                        {
+                            return topicList.IndexOf(a.Name) -
+                                topicList.IndexOf(b.Name);
+                        }
+                    );
+                    Assert.Empty(ex.Results.TopicDescriptions[0].AuthorizedOperations);
+                    Assert.Empty(ex.Results.TopicDescriptions[1].AuthorizedOperations);
+                    Assert.True(ex.Results.TopicDescriptions[0].Error.IsError);
+                    Assert.True(!ex.Results.TopicDescriptions[1].Error.IsError);
+                }
+
+                var topicListAuthOps = 
+                TopicCollection.OfTopicNames(
+                    new List<string>
+                    {
+                        singlePartitionTopic
+                    }
+                );
+                var descResWithAuthOps = await adminClient.DescribeTopicsAsync(
+                    topicListAuthOps,
+                    describeOptionsWithAuthOps);
+                Assert.NotEmpty(descResWithAuthOps.TopicDescriptions[0].AuthorizedOperations);
+                Assert.Equal(8, descResWithAuthOps.TopicDescriptions[0].AuthorizedOperations.Count);
+
+                var topicACLs =  new List<AclBinding>
+                {
+                    new AclBinding()
+                    {
+                        Pattern = new ResourcePattern
+                        {
+                            Type = ResourceType.Topic,
+                            Name = singlePartitionTopic,
+                            ResourcePatternType = ResourcePatternType.Literal
+                        },
+                        Entry = new AccessControlEntry
+                        {
+                            Principal = "User:user",
```

Comment:
Use 
"User:(value of string user)"
rather than hard-coding it.
For all the relevant tests.